src/iop/lowlight.c

```diff
@@ -625,33 +625,42 @@ static gboolean lowlight_draw(GtkWidget *widget, cairo_t *crf, gpointer user_dat
   cairo_set_operator(cr, CAIRO_OPERATOR_SOURCE);
 
   // draw labels:
-  cairo_text_extents_t ext;
+  PangoLayout *layout;
+  PangoRectangle ink;
+  PangoFontDescription *desc = pango_font_description_from_string("sans-serif bold");
+  pango_font_description_set_absolute_size(desc,(.06 * height) * PANGO_SCALE);
+  layout = pango_cairo_create_layout(cr);
   cairo_set_source_rgb(cr, .1, .1, .1);
-  cairo_select_font_face(cr, "sans-serif", CAIRO_FONT_SLANT_NORMAL, CAIRO_FONT_WEIGHT_BOLD);
   cairo_set_font_size(cr, .06 * height);
 
-  cairo_text_extents(cr, _("dark"), &ext);
-  cairo_move_to(cr, .02 * width + ext.height, .5 * (height + ext.width));
+  pango_layout_set_text(layout, _("dark"), -1);
+  pango_layout_set_font_description(layout, desc);
+  pango_layout_get_pixel_extents(layout, &ink, NULL);
+  double y = .5 * (height + ink.width);
+  double x = .02 * width + ink.height;
+  cairo_move_to(cr, x- ink.height , y -ink.y );
```

Comment:
Why the two variables `x` and `y`? And why the `+ ink.height` in `x` if you subtract it again?